cmd/whitesourceExecuteScan.go

```diff
@@ -388,7 +388,7 @@ func validateProductVersion(version string) string {
 func wsScanOptions(config *ScanOptions) *ws.ScanOptions {
 	return &ws.ScanOptions{
 		BuildTool:                  config.BuildTool,
-		ScanType:                   config.ScanType,
+		ScanType:                   "", // no longer provided via config
```

Comment:
Is this still needed or just kept to avoid major refactoring for now?

Reply:
just done to avoid further refactoring for now ...